### ISSUE TITLE
Study 'entry page' - you are now leaving content made by Lookit, starting content made by the researcher. 

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1058]
# TL;DR
(Counterpart to #1054)

# Narrative

As a platform, we want to be really clear when the content provided by the researcher is taking over from the central Lookit experience - for participants, this should cue me to contact the *researcher*, not the Lookit platform, if I am unhappy with something that happened in the study.

<details>
<summary>Feature Request fields</summary>
<p>
**With this `details` wrapper, you can just use this ticket to record your findings.
When you're ready to turn this into an implementation request, delete the scoping label and
this wrapper, leaving only the filled out information.**

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)

</p>
</details>
